ooniprobe/View/Informed consent/PopQuizViewController.m

```diff
@@ -0,0 +1,108 @@
+#import "PopQuizViewController.h"
+
+@interface PopQuizViewController ()
+
+@end
+
+@implementation PopQuizViewController
+@synthesize question_number;
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.trueButton addTarget:self action:@selector(answer:) forControlEvents:UIControlEventTouchUpInside];
+    [self.trueButton setTitle:[NSLocalizedString(@"_true", nil) uppercaseString] forState:UIControlStateNormal];
+    
+    [self.falseButton addTarget:self action:@selector(answer:) forControlEvents:UIControlEventTouchUpInside];
+    [self.falseButton setTitle:[NSLocalizedString(@"_false", nil) uppercaseString] forState:UIControlStateNormal];
+    
+    [self.closeView addTarget:self action:@selector(dismissPopup) forControlEvents:UIControlEventTouchUpInside];
+    
+    self.cointainerWindow.layer.cornerRadius = 12;
+    self.cointainerWindow.layer.masksToBounds = true;
+    
+    [self.titleLabel setText:NSLocalizedString(@"pop_quiz", nil)];
+    [self reloadQuestion];
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+- (void)reloadQuestion{
+    NSMutableAttributedString *question_intro = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"%@ %ld/2\n\n", NSLocalizedString(@"question", nil), question_number]];
```

Comment:
I would really appreciate if you could use Xcode to wrap long lines. It is very painful to review pull requests with lines longer than 80 chars and it is making me less effective and less happy to do reviews.